app/views/planning_applications/validation/environment_impact_assessments/_form.html.erb

```diff
@@ -1,39 +1,109 @@
-<%= form_with model: [@planning_application], url: planning_application_validation_environment_impact_assessment_url(@planning_application) do |form| %>
-  <%= form.govuk_error_summary(presenter: ErrorPresenter) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: [@environment_impact_assessment], url: planning_application_validation_environment_impact_assessment_url(@planning_application) do |form| %>
+      <% if @environment_impact_assessment.errors.any? %>
```

Comment:
Instead of this block does just adding 
```
  <%= form.govuk_error_summary(presenter: ErrorPresenter) %>
  ```
  do the same trick?

Reply:
Do you know how to do the different title errors vs the errors that get shown on the form? That's why I've got this separate block here

Reply:
Not off the top of my head, maybe just go ahead and merge and I can attempt as part of my PR